ci/code-quality/whitespace-lints.sh

```diff
@@ -37,6 +41,10 @@ exit_code=0
 
 check_code_quality -nHIP "\s+$" "found: trailing whitespaces" || exit_code=1
 check_code_quality -nHIP $"\t" "found: tabs" || exit_code=1
-check_code_quality -zLIP ".*\n\Z" "not found: newline at EOF" || exit_code=1
+check_code_quality -zLIP ".*\n\Z" "found: newline at EOF" || exit_code=1
```

Comment:
actually, this will print the files that has **no newline** at EOF :sweat_smile: